packages/core/contracts/cross-chain-oracle/chain-adapters/Optimism_ParentMessenger.sol

```diff
@@ -51,7 +84,8 @@ contract Optimism_ParentMessenger is OVM_CrossDomainEnabled, ParentMessengerInte
         bytes memory dataSentToChild =
             abi.encodeWithSignature("processMessageFromCrossChainParent(bytes,address)", data, target);
         sendCrossDomainMessage(childMessenger, defaultGasLimit, dataSentToChild);
-        emit MessageSentToChild(data, target, defaultGasLimit);
+
+        emit MessageSentToChild(dataSentToChild, target, defaultGasLimit);
```

Comment:
Can you revert change to this event? I clean up events and make them more detailed in #3620 and want to reduce merge conflicts

Reply:
done.